core/commands/pin/pin.go

```diff
@@ -362,6 +362,7 @@ Example:
 		cmds.BoolOption(pinQuietOptionName, "q", "Write just hashes of objects."),
 		cmds.BoolOption(pinStreamOptionName, "s", "Enable streaming of pins as they are discovered."),
 		cmds.BoolOption(pinNamesOptionName, "n", "Enable displaying pin names (slower)."),
+		cmds.StringOption(pinNameOptionName, "Display only pins with the given name."),
```

Comment:
This would align it more with what is said in the initial issue:

```suggestion
		cmds.StringOption(pinNameOptionName, "Display pins with names that contain the value provided (case-sensitive, exact match)."),
```

Reply:
@lidel done